test/test_support.ts

```diff
@@ -34,26 +37,20 @@ const {cachedLibPath, cachedLib} = (function() {
   let host = ts.createCompilerHost(compilerOptions);
   let fn = host.getDefaultLibFileName(compilerOptions);
   let p = ts.getDefaultLibFilePath(compilerOptions);
-  return {cachedLibPath: p, cachedLib: host.getSourceFile(fn, ts.ScriptTarget.ES2015)};
+  return {
+    cachedLibPath: path.normalize(p),
+    cachedLib: host.getSourceFile(fn, ts.ScriptTarget.ES2015)
+  };
 })();
 
 /** Creates a ts.Program from a set of input files. */
 export function createProgram(sources: Map<string, string>): ts.Program {
   let host = ts.createCompilerHost(compilerOptions);
 
-  // Fake out host.directoryExists so that it doesn't read through node_modules/@types.
-  let realDirectoryExists = host.directoryExists;
-  host.directoryExists = dirName => {
-    if (path.isAbsolute(dirName)) {
-      let relName = path.relative(process.cwd(), dirName);
-      if (relName === 'node_modules/@types') return false;
-    }
-    return realDirectoryExists ? realDirectoryExists(dirName) : false;
-  };
-
   host.getSourceFile = function(
                            fileName: string, languageVersion: ts.ScriptTarget,
                            onError?: (msg: string) => void): ts.SourceFile {
+    fileName = path.normalize(fileName);
```

Comment:
Yes, `cachedLibPath` has mixed directory seperators on windows, e.g. `tsickle\node_modules\typescript\lib/lib.es6.d.ts`, and `fileName` has only unix like directory seperators `tsickle/node_modules/typescript/lib/lib.es6.d.ts`. That's why the equality check in the next line fails.

I think it's worth explaining and add a comment.